src/components/v2/Toast/index.tsx

```diff
@@ -0,0 +1,73 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import { toast as toastify, ToastOptions } from 'react-toastify';
+import { GlobalStyles } from '@mui/material';
+
+import { Notice } from '../Notice';
+import { Icon } from '../Icon';
+import { Button } from '../Button';
+
+import 'react-toastify/dist/ReactToastify.min.css';
+import { customToastGlobalStyles, useStyles } from './styles';
+
+interface IToastArgs {
+  title: string;
+  description?: string;
```

Comment:
According to the new designs, notifications don't have titles anymore:
<img width="408" alt="Screen Shot 2022-05-18 at 08 46 48" src="https://user-images.githubusercontent.com/44675210/168985787-9eed1bb0-9b89-40c3-864a-9f3e77a9b341.png">

Reply:
removed titles for notice component and for toast notifications

Reply:
done